packages/blocks/table/src/mockData/mockData.ts

```diff
@@ -116,34 +113,71 @@ const people: Person[] = [
 ];
 
 export const initialTableData = {
-  initialState: {
-    hiddenColumns: [
-      "entityId",
-      "employerId",
-      "employer.locationId",
-      "employer.entityId",
-      "employer.type",
-      "employer.location.entityId",
-      "employer.location.type",
-    ],
-  } as TableOptions<{}>["initialState"] & {
-    columns?: { Header: string; accessor: string }[];
-  },
   data: {
     __linkedData: {
-      entityTypeId: "Person",
       aggregate: {
-        itemsPerPage: 3,
-        multiSort: [
-          {
-            field: "name",
-          },
-        ],
+        itemsPerPage: 5,
```

Comment:
(Maybe) Unrelated Change: Updating the mock data for the table block to match the data present in the hub demo.

The previous mock data was broken so this change is needed to test the block well.